pkg/apiserver/statement/queries.go

```diff
@@ -65,7 +83,8 @@ func QueryStatementsOverview(db *gorm.DB, schemas []string, beginTime, endTime s
 			sum(exec_count) AS agg_exec_count,
 			round(sum(exec_count*avg_affected_rows)/sum(exec_count)) AS agg_avg_affected_rows,
 			round(sum(exec_count*avg_latency)/sum(exec_count)) AS agg_avg_latency,
-			round(sum(exec_count*avg_mem)/sum(exec_count)) AS agg_avg_mem
+			round(sum(exec_count*avg_mem)/sum(exec_count)) AS agg_avg_mem,
+			group_concat(table_names) AS agg_schemas
```

Comment:
I prefer to use [`AfterFind` hook](http://gorm.io/docs/hooks.html#Querying-an-object)

Reply:
good idea! will update it soon.